### ISSUE TITLE
Lines in docblock are removed if they follow a removed docblock "annotation"

### ISSUE THREAD

**Issue by jaydiablo** [FriendsOfPHP/PHP-CS-Fixer#1918]
I looked through the tickets a bit to see if this has been reported before, but it didn't seem like it has.

I'm using the current version of php-cs-fixer (which is 1.11.2 as of this writing), with the following rules (from our .php_cs file):

```php
return Symfony\CS\Config\Config::create()
    ->fixers(['-concat_without_spaces', '-single_quote', '-unalign_double_arrow', '-unalign_equals', 'short_array_syntax', 'align_equals', 'align_double_arrow'])
    ->setUsingCache(true)
    ->finder($finder)
;
```

We ran the fixer on a wordpress theme, which had some docblocks like this:

```php
<?php
/**
 * The Template for displaying basic pages.
 *
 * @package Wordpress
 * @subpackage OurTheme
 *
 * Template Name: Landing Page (full width)
 */
```

After php-cs-fixer ran, the file looked like this:

```php
<?php
/**
 * The Template for displaying basic pages.
 */
```

Unfortunately, in Wordpress, the line that begins with "Template Name:" is important, as it's how Wordpress determines if the file is a template that should be selectable in the admin UI.

I believe this is happening because of the removal of the `@package` and `@subpackage` annotations.  If I add back in the "Template Name:" line (with the `@package` and `@subpackage` annotations removed), php-cs-fixer doesn't touch the "Template Name:" line at all.

Is this intended behavior (to remove all the lines that follow the `@subpackage` annotation)?

One of the files it ran against had this docblock:

```php
<?php
/**
 * This page template is for our legacy site
 *
 * @package Wordpress
 * @subpackage OurTheme
 * @since
 *
 * Template Name: Legacy Home
```

When php-cs-fixer is run against it, this is the result:

```php
<?php
/**
 * This page template is for our legacy site
 *
 * @since
 *
 * Template Name: Legacy Home
```

The `@since` annotation prevents the line from being removed.  Perhaps the code that removes the `@subpackage` line is a bit over-zealous?

**Comment by keradus** [comment]
It looks like You have wrong PHPDoc. Whole PHP CS Fixer fixers for PHPDoc follows PHPDoc standard (/ PSR-5 proposal)

Your docblock should look like:
```php
/**
 * Summary.
 *
 * Description.
 * Could Be multilined
 *
 * @annotationA
 * @annotationB with description
 *
 * @annotationC with description that is
 * multiline
 */
```

For that, `Template Name: ...` is treaten as annotation description.
Move your description after summary and before annotations section, and it should work

**Comment by GrahamCampbell** [comment]
Indeed this is not a bug, Your last part is parsed by most phpdoc parsers as part of the description for the annotation above it. I found many cases in places like guzzle and the aws sdk where that style of parsing is assumed when I originally researched this.

TLDR: All phpdoc level descriptions must go before annotations, otherwise they become attached to the annotation above them.

**Comment by jaydiablo** [comment]
Ok, thanks for the info, I'll be sure that our team doesn't write doc blocks in that fashion from now on :)
